resources/views/profile/logout-other-browser-sessions-form.blade.php

```diff
@@ -1,99 +1,182 @@
-<x-jet-action-section>
-    {{-- TODO: rewrite to our components --}}
-    <x-slot name="title">
-        @lang('ui::forms.logout-sessions.title')
-    </x-slot>
-
-    <x-slot name="description">
-        @lang('ui::forms.logout-sessions.description')
-    </x-slot>
-
-    <x-slot name="content">
-        <div class="max-w-xl text-sm text-gray-600">
+<div dusk="logout-other-browser-sessions">
+    <div class="flex flex-col">
+        <span class="header-4">@lang('ui::forms.logout-sessions.title')</span>
+        <span class="mt-4">@lang('ui::forms.logout-sessions.description')</span>
```

Comment:
the other settings components use `ui::pages.user-settings.<section>` , better to keep the translations all in one place

Reply:
Done